frontend/src/routes/Dashboard/styles.module.css

```diff
@@ -85,25 +71,55 @@
     bottom: 2rem;
 }
 
+.header {
+    text-align: left;
+    right: 3rem;
+}
+
+.header h3{
+    margin-block-end: 0.5em;
```

Comment:
I'd suggest using `margin-bottom` and lowering it to `0.2rem`. Always prefer `rem` to `em`!

`em` vary depending their location on the dom tree & surrounding rules. `rem` vary according to the `font-size` given to `html`, which gives us more control.